contracts/utils/ETHOSMedianizer.sol

```diff
@@ -12,7 +12,7 @@ interface OracleSecurityModule {
     function bud(address) external view returns (uint256);
 }
```

Comment:
So, just move `interface OracleSecurityModule` to `interfaces/maker/OracleSecurityModule.sol` and we're in business!